README.md

```diff
@@ -4,23 +4,10 @@
 
 ## Use in your project
```

Comment:
Can you please add some description about:
- what is the purpose of this module (see [BOM](https://en.wikipedia.org/wiki/Bill_of_materials), it defines external dependencies versions)
- how to use it in implemented modules (Gradle / Maven)